contract_api/registry.py

```diff
@@ -217,7 +217,7 @@ def get_group_info(self, org_id, service_id):
                                          "free_call_signer_address": rec.get("free_call_signer_address", "")
                                          }
                               }
-                    groups[group_id]['endpoints'].append({"endpoint": rec['endpoint'], "is_available":
+                groups[group_id]['endpoints'].append({"endpoint": rec['endpoint'], "is_available":
```

Comment:
Does this mean we will now return multiple end-points ..as against one previously sent ? this has a direct impact on Marketplace dapp no ? lets review with @Vivek205 on any impact